docs/debug-functions.md

```diff
@@ -36,9 +36,18 @@ $ kubeless function ls
 NAME 	NAMESPACE	HANDLER  	RUNTIME  	DEPENDENCIES	STATUS
 foo  	default  	hello,foo	python3.6	            	MISSING: Check controller logs
 $ kubectl logs -n kubeless -l kubeless=controller
-time="2018-04-27T15:12:28Z" level=info msg="Processing update to function object foo Namespace: default" controller=cronjob-trigger-controller
-time="2018-04-27T15:12:28Z" level=error msg="Function can not be created/updated: failed: incorrect handler format. It should be module_name.handler_name" pkg=function-controller
-time="2018-04-27T15:12:28Z" level=error msg="Error processing default/foo (will retry): failed: incorrect handler format. It should be module_name.handler_name" pkg=function-controller
+Error from server (BadRequest): a container name must be specified for pod kubeless-controller-manager-cd68f56c4-cjbnz, choose one of: [kubeless-function-controller http-trigger-controller cronjob-trigger-controller]
+$ kubectl logs -n kubeless -l kubeless=controller -c kubeless-function-controller
+time="2020-10-01T01:48:29Z" level=info msg="Processing change to Function default/foo" pkg=function-controller
+time="2020-10-01T01:48:29Z" level=error msg="Function can not be created/updated: failed: incorrect handler format. It should be module_name.handler_name" pkg=function-controller
```

Comment:
Can you delete the first `kubectl logs`? It's true that it requires now to use a container name. Also, from all the log lines you added, the only two important are the first two, can you remove the rest?

Reply:
ok，I have removed the first `kubectl logs`.